File: trunk/Templates/Examples/NHibernate/CSharp/NHibCsSample.Data/ManagerObjects/AccountManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using NHibCsSample.Generated.BusinessObjects;
using NHibCsSample.Generated.Base;

namespace NHibCsSample.Generated.ManagerObjects
{
    public partial interface IAccountManager : IManagerBase<Account, int>
    {
	}
	
	partial class AccountManager : ManagerBase<Account, int>, IAccountManager
    {
	}
}

File: Templates/Examples/NHibernate/VisualBasic/NHibVbSample.Data/ManagerObjects/AccountManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports NHibVbSample.Generated.BusinessObjects
Imports NHibVbSample.Generated.Base

Namespace NHibVbSample.Generated.ManagerObjects
	'Public Partial Interface IAccountManager
	'	Inherits IManagerBase(Of Account, System.Int32)
	'End Interface

	Partial Class AccountManager
		Inherits ManagerBase(Of Account, System.Int32)
		Implements IAccountManager
	End Class
End Namespace
